docker/build_scripts/build.sh

```diff
@@ -48,9 +48,10 @@ cp $MY_DIR/epel-release-5-4.noarch.rpm .
 check_sha256sum epel-release-5-4.noarch.rpm $EPEL_RPM_HASH
 
 # Dev toolset (for LLVM and other projects requiring C++11 support)
-wget -q http://people.centos.org/tru/devtools-2/devtools-2.repo
+wget -q https://people.centos.org/tru/devtools-2/devtools-2.repo
 check_sha256sum devtools-2.repo $DEVTOOLS_HASH
 mv devtools-2.repo /etc/yum.repos.d/devtools-2.repo
+sed -i 's/http/https/g' /etc/yum.repos.d/devtools-2.repo
```

Comment:
can you make sure `https` won't be transformed as `httpss` in case they also make the change upstream on this one ?